nav2_smac_planner/src/smac_planner.cpp

```diff
@@ -111,8 +111,8 @@ void SmacPlanner::configure(
   node->get_parameter(name + ".analytic_expansion_ratio", search_info.analytic_expansion_ratio);
 
   nav2_util::declare_parameter_if_not_declared(
-    node, name + ".max_planning_time_ms", rclcpp::ParameterValue(5000.0));
-  node->get_parameter(name + ".max_planning_time_ms", _max_planning_time);
+    node, name + ".max_planning_time", rclcpp::ParameterValue(5.0));
```

Comment:
I see you changed the units of the parameter, but you don't make any updates in the code itself where it is assumed to be in `ms` right now?

Reply:
The only place it's used is [here](https://github.com/ros-planning/navigation2/blob/3bbb4fe020194f2d866ebe50a2ae82e43e260655/nav2_smac_planner/src/smac_planner.cpp#L328). I've tested similar peace of code in a sandbox: 
```#include <iostream>
#include <chrono>
#include <thread>

using namespace std::chrono_literals;
using namespace std::chrono;

int main(int argc, char *argv[])
{
	double _max_planning_time = 5.0;
	steady_clock::time_point a = steady_clock::now();
	std::this_thread::sleep_for(100ms);
	steady_clock::time_point b = steady_clock::now();
	duration<double> time_span = duration_cast<duration<double>>(b - a);
	double time_remaining = _max_planning_time - static_cast<double>(time_span.count()); // TODO
	std::cout << time_remaining << '\n';
	std::cout << static_cast<double>(time_span.count()) << '\n';
	return 0;
}
```
Output:
```
4.89993
0.100073
```
That ```time_span``` variable is already in seconds. I guess the previous implementation was flawed actually.

Reply:
Ah got it, thanks